scenario.py

```diff
@@ -149,6 +149,14 @@ def __init__(self, params, experiment_path, scenario_id=1, n_repeat=1, is_dry_ru
         else:
             self.is_early_stopping = True  # default
 
+        # Model from previous coalition used to initialise model
+        self.use_weights_from_previous_coalition = False
+        try:
+            self.init_model_from = params["init_model_from"]
+            self.use_weights_from_previous_coalition = True
+        except:
+            self.init_model_from= ""
```

Comment:
Is there an interest in setting it to an empty string? Couldn't we just leave it undefined? (`pass`...)